### PR TITLE
Add rspec-otel to registry

### PR DESCRIPTION
**Pull request by dmathieu**
This adds the rspec-otel gem to the registry.
https://rubygems.org/gems/rspec-otel

### CODE REVIEW

File: data/registry/tools-ruby-rspec-matcher.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+language: ruby
+tags:
+  - ruby
+  - rspec
+license: MIT
+description:
+  The RSpec matchers tool provides a set of matchers to make it easier to write
+  unit tests checking the presence of telemetry data.
+authors:
+  - name: Damien MATHIEU # cspell:disable-line
```
Comment (svrnm) [inline data/registry/tools-ruby-rspec-matcher.yml:12]:
```suggestion
  - name: Damien Mathieu
```
@dmathieu as suggested move the cspell ignore to the top of the file. Also if you don't mind I would prefer to not uppercase your last name (except this is how you commonly write it, then ignore that suggestion)

Comment (dmathieu) [reply data/registry/tools-ruby-rspec-matcher.yml]: I usually write it in full uppercase, to show which is my name and which is my last name. But not doing so is fine too.

**Comment by dmathieu** [comment]
I suppose the repo URL needs to be added to refcache.
But I don't see any documentation about doing so in the repository, and my attempt to add it manually don't resolve the failure.

**Comment by svrnm** [comment]
> I suppose the repo URL needs to be added to refcache. But I don't see any documentation about doing so in the repository, and my attempt to add it manually don't resolve the failure.

yes, we need to document this better in our contributor guidelines

**Pull request merged**